.gitignore

```diff
@@ -18,4 +18,6 @@ build
 dist
 .idea
 toolchain.tar.gz
-time.json
\ No newline at end of file
+time.json
+*.pyc
```

Comment:
because we have python code, the cache-related files and folders should be omitted